src/api/app/controllers/webui/project_controller.rb

```diff
@@ -779,7 +779,7 @@ def find_maintenance_infos
 
     @is_maintenance_project = @project.is_maintenance?
     if @is_maintenance_project
-      @open_maintenance_incidents = @project.maintenance_incidents.pluck('projects.name').sort.uniq
+      @open_maintenance_incidents = @project.maintenance_incidents.pluck('projects.name').sort!.uniq!
```

Comment:
There are some cases were the 'bang' version is returning nil when the object wasn't changed, [uniq](https://ruby-doc.org/core-2.5.0/Array.html#method-i-uniq-21) is one of them.

Reply:
yeah, mean pitfall - will have to revert some cases :)